### PR TITLE
master for Blender 2.83 Nightly builds

### PR DESCRIPTION
**Pull request by zeffii**
as per https://github.com/nortikin/sverchok/issues/2891  Read it for details

TL;DR:

The signature of EnumProperties (used for modes/presets/selectable items from a list) are used pervasively throughout Sverchok and many add-ons. Until recently the code definitions did not have strict rules for the string associated with each enum item (called "Identifier") . Now Blender enforces a strict policy that each identifier must contain no spaces. If you try to load a node that we haven't fixed yet, it will crash blender.

```
old allowed
[("Identifier A", "name", "description", icon, index), .... ]

must now be
[("Identifier_A", "name", "description", icon, index), .... ]

```

error:
## enum identifiers must not contain spaces.
afflicted:

- [x] SvTextureViewerNodeLite `.output_mode`
- [x] SvMatrixTrackToNode (`.tu_axes, .tu_mapping`)
- [x] SvScriptNodeLite `.selected_mode`
- [x] SvGetAssetProperties `.gp_selected_frame_mode`
- [x] SvIntersectEdgesNodeMK2 `.alg_mode_2d`
- [x] SvListModifierNode. `func_`
- [x] SvBMOpsNodeMK2 `.oper`
- [x] SvTextureEvaluateNode `.color_channel`
- [x] SvCastNode `.cast_mode`
- [x] SvFormulaShapeNode (`.formulaX , .formulaY,  .formulaZ`)
- [x] SvContourNode `.list_match`
- [x] SvPointInside `.selected_algo`
- [x] SvCropMesh2D `.alg_mode`
- [x] SvMergeMesh2DLite `.alg_mode`
- [x] SvSetCustomMeshNormals `.mode`
- [x] SvVectorMathNodeMK3 `.current_op`
- [x] SvScalarMathNodeMK4 `.current_op`
- [x] SvDisplaceNode (`.out_mode, .color_channel, .tex_coord_type`)  <--Texture Displace
- [x] SvComponentAnalyzerNode (`.vertex_mode, .edge_mode, .face_mode, .tangent_mode, .center_mode, .pols_origin_mode`)
- [x] SvSuperEllipsoidNode `.presets`
- [x] SvSpiralNode `.presets`
- [ ] SvListStatisticsNode `.function`
- [ ] SvTransformMesh (`.mask_mode, .origin_mode`)

error:
##  default is not in items.
afflicted:
- [x] SvTextureViewerNodeLite `.output_mode`
- [x] SvScriptNodeLite `.selected_mode`
- [x] SvVectorMathNodeMK3 `.current_op`
- [x] SvDisplaceNode `.tex_coord_type`
- [x] SvComponentAnalyzerNode (`.edge_mode, .center_mode, .pols_origin_mode`)

### CODE REVIEW

**Comment by zeffii** [comment]
I may have broken FormulaShape node a little. sorry. will return to later.

**Comment by zeffii** [comment]
ok. back to this nonsense.

**Comment by zeffii** [comment]
@vicdoval @DolphinDream @portnov 
the remaining nodes you will have to fix yourselves.

**Comment by zeffii** [comment]
i've merged recent Master additions into this branch.
